src/components/cards/CardMenu.vue

```diff
@@ -126,9 +126,7 @@ export default {
 			return true
 		},
 		boards() {
-			return this.$store.getters.boards.filter(board => {
-				return board.id !== this.currentBoard.id
-			})
+			return this.$store.getters.boards
```

Comment:
Sorry missed that in the first review, we can actually just add it to use mapGetters in line 108